pkg/volume/plugins.go

```diff
@@ -384,6 +391,36 @@ func (spec *Spec) Name() string {
 	}
 }
 
+// IsControllerExpandable returns true for volume types that can be expanded by the controller
+// and false for those that need to be run on the node. Flex volume is the only one in the
+// latter category since its typically not installed on the controller/master
+func (spec *Spec) IsControllerExpandable() bool {
+	switch {
+	case spec.Volume != nil:
+		return spec.Volume.FlexVolume == nil
+	case spec.PersistentVolume != nil:
+		return spec.PersistentVolume.Spec.FlexVolume == nil
+	default:
+		return true
```

Comment:
return false for default case?

Reply:
Fair enough - although in such a case (i.e. `.Volume == nil && .PersistentVolume == nil`) something else altogether would be wrong.